src/main/java/frc/robot/commands/DriveToNoteWithSearch.java

```diff
@@ -0,0 +1,55 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.chassis.Chassis;
+
+public class DriveToNoteWithSearch extends Command {
+  private final Chassis chassis;
+
+  NetworkTable table = NetworkTableInstance.getDefault().getTable("limelight");
+  NetworkTableEntry tx = table.getEntry("tx");
+  NetworkTableEntry ty = table.getEntry("ty");
+  NetworkTableEntry tv = table.getEntry("tv"); // 1 if target is deteced, 0 if not.
+
+  private final PIDController translationController = new PIDController(0.07, 0, 0);
+  private final PIDController rotationController = new PIDController(0.07, 0, 0);
+  private final double searchRotationSpeed = 0.07;
+
+  /** Creates a new FieldOrientedDrive. */
+  public DriveToNoteWithSearch(Chassis chassis) {
+    this.chassis = chassis;
+
+    translationController.setSetpoint(0.0);
+    rotationController.setSetpoint(0.0);
+
+    addRequirements(chassis);
+  }
+
+  @Override
+  public void initialize() {
+    translationController.reset();
+    rotationController.reset();
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    double dv = tv.getDouble(0.0);
```

Comment:
We used `d` in the other two variables to represent delta, or change. Since this is just yes or no, we should give this a more descriptive name. We could call it 

`noteDetected`

Also, we could check its value here and make the type a boolean

`boolean noteDetected = tv.getDouble(0.0) == 1.0`